verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -0,0 +1,88 @@
+import { Profile, Strategy, VerifyCallback } from 'passport-google-oauth20';
+import { Request, Response } from 'express';
+import { User, UserDocument } from '../model/user';
+
+import passport from 'passport';
+
+export const router = require('express').Router();
+
+router.get(
+    '/google/redirect',
+    passport.authenticate('google'),
```

Comment:
Could you add a comment describing what this call does?

Reply:
done.